asyncy/hub/sdk/AsyncyHub.py

```diff
@@ -98,6 +98,10 @@ def get(self, alias=None, owner=None, name=None) -> Service:
 
     def _get(self, alias: str = None, owner: str = None, name: str = None):
         try:
+            if '/' in alias and alias.count("/") == 1:
```

Comment:
```suggestion
            if alias is not None and alias.count("/") == 1:
```

No need to be redundant.

Reply:
on it!